root/edit/details/AddArea.js

```diff
@@ -27,7 +27,7 @@ type AddAreaEditT = {
     +iso_3166_3: $ReadOnlyArray<string>,
     +name: string,
     +sort_name: string | null,
-    +type: AreaTypeT,
+    +type: AreaTypeT | null,
```

Comment:
I guess this isn't new, but an old typing error? :)

Reply:
Yeah. I'll go back and try to split out some of the changes you've brought up into separate commits.